app/scripts/background.js

```diff
@@ -170,7 +170,7 @@ if (isManifestV3) {
 async function initialize(remotePort) {
   const initState = await loadStateFromPersistence();
   const initLangCode = await getFirstPreferredLangCode();
-  await setupController(initState, initLangCode, remotePort);
```

Comment:
Nit: This function does seem to return a Promise, despite being fully synchronous. It returns `Promise.resolve()` at the end. We should probably remove the `return Promise.resolve()` and update the JSDoc comment if we're removing this `await`.